scripts/Utility.js

```diff
@@ -17,4 +17,24 @@ function formatHour(time) {
 		return time + 'a';
 	}
 	return (time - 12) + 'p';
-}
\ No newline at end of file
+}
+
+function getHolidays() {
```

Comment:
Good point. I wish I'd known about that; that's definitely a better song. I'll probably get rid of the KK holiday stuff then, and add a separate option for holiday songs like that.